doc/source/user_guide/basics.rst

```diff
@@ -1022,7 +1022,7 @@ Mixed dtypes
 ++++++++++++
 
 When presented with mixed dtypes that cannot aggregate, ``.agg`` will only take the valid
-aggregations. This is similar to how groupby ``.agg`` works.
```

Comment:
This is correct I believe. I think it's referring to the `groupby.agg` functionality

Reply:
I see, I was not aware of that function.